packages/epics/src/membership/components/inner-spaces/inner-spaces-list.tsx

```diff
@@ -0,0 +1,39 @@
+import { FC } from 'react';
+import { InnerSpaceCard } from './inner-space-card';
+import { useInnerSpaces } from '../../hooks/use-inner-spaces';
+
+type InnerSpacesListProps = {
+  page: number;
+  activeSort: string;
+};
+
+export const InnerSpacesList: FC<InnerSpacesListProps> = ({
+  page,
+  activeSort,
+}) => {
+  const { innerSpaces, isLoading } = useInnerSpaces({
+    page,
+    sort:
+      activeSort === 'all'
+        ? { sort: 'all' }
+        : activeSort === 'most-recent'
+        ? { sort: 'most-recent' }
+        : undefined,
+  });
```

Comment:
Let's refactor like this first:

```suggestion
import { SortParams } from '@hypha-platform/graphql/rsc';

type InnerSpacesListProps = {
  page: number;
  activeSort?: SortParams['sort'];
};

export const InnerSpacesList: FC<InnerSpacesListProps> = ({
  page,
  activeSort,
}) => {
  const { innerSpaces, isLoading } = useInnerSpaces({
    page,
    sort: { sort: activeSort },
  });
```